starters/lit-mobx-vaadin/package.json

```diff
@@ -0,0 +1,85 @@
+{
+	"name": "lit-mobx-vaadin",
+	"description": "Lit, Mobx, and Vaadin",
+	"hasShowcase": false,
+	"keywords": [
+		"lit",
+		"mobx",
+		"vaadin"
+	],
+	"version": "0.1.0",
+	"type": "module",
+	"scripts": {
+		"lint": "eslint --ext .ts,.html . --ignore-path .gitignore && prettier \"**/*.ts\" --check --ignore-path .gitignore",
+		"prettier": "eslint --ext .ts,.html . --fix --ignore-path .gitignore && prettier \"**/*.ts\" --write --ignore-path .gitignore",
```

Comment:
Lets rename both `lint` and `prettier` scripts names to:

`format:check` and `format:write`